examples/chef/devices/rootnode_colortemperaturelight_hbUnzYVeyn.matter

```diff
@@ -0,0 +1,2215 @@
+    callback attribute packetUnicastRxCount;
+    callback attribute packetUnicastTxCount;
+    callback attribute currentMaxRate;
+    callback attribute overrunCount;
+    ram      attribute featureMap default = 3;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster EthernetNetworkDiagnostics {
+    callback attribute PHYRate;
+    callback attribute fullDuplex;
+    callback attribute packetRxCount;
+    callback attribute packetTxCount;
+    callback attribute txErrCount;
+    callback attribute collisionCount;
+    callback attribute overrunCount;
+    callback attribute carrierDetect;
+    callback attribute timeSinceReset;
+    ram      attribute featureMap default = 3;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster Switch {
+    ram      attribute numberOfPositions default = 2;
+    ram      attribute currentPosition;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 1;
+  }
+
+  server cluster AdministratorCommissioning {
+    callback attribute windowStatus;
+    callback attribute adminFabricIndex default = 1;
+    callback attribute adminVendorId;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster OperationalCredentials {
+    callback attribute NOCs;
+    callback attribute fabrics;
+    callback attribute supportedFabrics;
+    callback attribute commissionedFabrics;
+    callback attribute trustedRootCertificates;
+    callback attribute currentFabricIndex;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 0x0001;
+  }
+
+  server cluster GroupKeyManagement {
+    callback attribute groupKeyMap;
+    callback attribute groupTable;
+    callback attribute maxGroupsPerFabric;
+    callback attribute maxGroupKeysPerFabric;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 1;
+  }
+
+  server cluster FixedLabel {
+    callback attribute labelList;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 1;
+  }
+
+  server cluster UserLabel {
+    callback attribute labelList;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 1;
+  }
+}
+endpoint 1 {
+  device type anonymousEndpointType = 268;
+
+  server cluster Identify {
+    ram      attribute identifyTime;
+    ram      attribute identifyType;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 4;
+  }
+
+  server cluster Groups {
+    ram      attribute nameSupport;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap;
+    ram      attribute clusterRevision default = 3;
+  }
+
+  server cluster Scenes {
```

Comment:
Is scenes supposed to be here? I think 1.0 does not support scenes

Reply:
Really? Can you help to share more context about this?

On the other hand, the Scenes are already enabled in many devices
```
$ grep "server cluster Scenes =" examples/chef/devices/*.matter
examples/chef/devices/noip_rootnode_dimmablelight_bCwGYSDpoe.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_colortemperaturelight_hbUnzYVeyn.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_dimmablelight_bCwGYSDpoe.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_doorlock_aNKYAreMXE.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_extendedcolorlight_8lcaaYJVAa.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_heatingcoolingunit_ncdGai1E5a.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_onofflight_bbs1b7IaOV.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_onofflightswitch_FsPlMr090Q.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_onoffpluginunit_Wtf8ss5EBY.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_thermostat_bm3fb8dhYi.matter:server cluster Scenes = 5 {
examples/chef/devices/rootnode_windowcovering_RLCxaGi9Yx.matter:server cluster Scenes = 5 {
```

Reply:
I did not code review the other examples ... overall I don't think the other examples are correct: #22640

So just because others have it does not make it correct.

Reply:
Hi @andy31415 , the Scene cluster has been removed. Please help to review again. Thanks.